src/main/java/com/linecorp/armeria/client/routing/ServerGroupRegistryHub.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.routing;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * A memory server group registry center.
+ */
+public class ServerGroupRegistryHub {
```

Comment:
We want ServerGroupRegistry as a group config center to expose public API to manage the whole system group config info,like query,register,update,refresh and so on.For example our admin system can use it to manage group info.

how about this? ServerGroupRegistry still public,and we can add delegate methods to ServerGroup which can make the client code more elegant and chaining style.